templates/frontend/objects/article_details.tpl

```diff
@@ -190,7 +190,18 @@
 			{/if}
 
 			{* References *}
-			{if $article->getCitations()}
+			{if $parsedCitations->getCount()}
+				<div class="item references">
+					<h3 class="label">
+						{translate key="submission.citations"}
+					</h3>
+					<div class="value">
+						{iterate from=parsedCitations item=parsedCitation}
+							<p>{$parsedCitation->getRawCitation()|escape}</p>
+						{/iterate}
+					</div>
+				</div>
+			{elseif $article->getCitations()}
```

Comment:
Everything is the same in the HTML exceept inside of `<div class="value">`, so maybe move the `if` conditional inside that.